### DISCUSSION TITLE
Adding existing snowflake objects

### DISCUSSION THREAD

**Discussion by datanrd** [theory/sqitch#667] [Q&A]
We have a snowflake deployment are looking at sqitch to source control and manage deployments. My question is how do you add the initial state to a sqitch project?

**Comment by theory** [comment]
From an existing database, you mean? Personally I have spent time writing changes for all my existing database objects, but if you have a lot of objects that can be daunting. Have a look at [this wiki](/sqitchers/sqitch/wiki/How-do-I-get-started-with-an-existing-database%3F) for a brief description of the steps to get things going with dump of database objects.
